src/TrackGenerator.h

```diff
@@ -94,6 +95,12 @@ class TrackGenerator {
   /** The z-coord where the 2D Tracks should be created */
   double _z_coord;
 
+  /** Determines the type of track segmentation to use */
+  segmentationType _segment_formation;
+
+  /** Max optical segment length for Tracks before splitting */
```

Comment:
would be more accurate to say "max optical path length"  than "max optical segment length" to match the term used in the literature